core/build.sbt

```diff
@@ -4,63 +4,6 @@ import Library._
 import sbtdocker.Instructions
 import sbtrelease.ReleaseStateTransformations._
 
-val javadocDisabledFor = Set(
```

Comment:
I move this down. Annoying to have it at the top. Need to scroll to the interesting parts and when doing a search we first hit the project names in this section.